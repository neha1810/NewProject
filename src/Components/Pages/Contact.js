import React, { Component } from 'react'
import banner from '../Images/banner.jpg'
class contact extends Component {
    render() {
        return (
            <fieldset>
                <div>
                    <div className="banner ">
                        <img className="col-xs-12 col-sm-8 col-md-8" id="banner" src={banner} alt="HPD_Friendshipday_1" />
                    </div>
                    <h1>Contact Details!</h1>

                </div>
            </fieldset>
        )
    }
}
export default contact;