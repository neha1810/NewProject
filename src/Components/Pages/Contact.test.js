import React from 'react';
import Contact from './Contact';
import {shallow} from 'enzyme';
// setup file
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('Contact', () => {
let wrapper;
beforeEach(()=> {
wrapper = shallow(
        <Contact />
    )
})


it('render the Contact component', () => {
    expect(wrapper).toMatchSnapshot();
});
it('Button exists',()=> {
    //wrapper.find('button').at(1).simulate('click');
    expect(wrapper.find('#Contact')).toBeDefined();
})



})