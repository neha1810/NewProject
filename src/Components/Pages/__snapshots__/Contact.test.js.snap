// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact render the Contact component 1`] = `ShallowWrapper {}`;
